zipline/assets/assets.py

```diff
@@ -156,18 +159,24 @@ def lookup_asset_types(self, sids):
         if not missing:
             return found
 
-        router_cols = self.asset_router.c
-        query = sa.select((router_cols.sid, router_cols.asset_type)).where(
-            self.asset_router.c.sid.in_(map(int, missing))
-        )
-        for sid, type_ in query.execute().fetchall():
-            missing.remove(sid)
-            found[sid] = self._asset_type_cache[sid] = type_
+        for chunk in self._group_into_chunks(list(missing)):
```

Comment:
For readability, what do you think about renaming `chunk` to `assets`, since the chunk is a list of assets?